src/main/ast/Regex.kt

```diff
@@ -0,0 +1,28 @@
+package ast
```

Comment:
I think that it's important that we remain consistent when naming packages in our DSL. I think the name `ast` is a good start, but maybe `dsl` might be more descriptive? We're more than laying out the structure of the language here - we're building what it is.

Reply:
True - I thought we might have a parser and interpreter somewhere else but they'll be built in to our AST nodes. I do think `dsl` might be overly vague? The whole project is our DSL.

Reply:
So far I have been using `dsl` in my local project. I say we stick with that and change it to whatever is appropriate in the big merge conflict at the end.